### ISSUE TITLE
Rendering issue with docs, converting double-hyphen to en dash

### ISSUE THREAD

**Issue by g-a-c** [borgbackup/borg#6343]
## Have you checked borgbackup docs, FAQ, and open Github issues?

Yes

## Is this a BUG / ISSUE report or a QUESTION?

Bug

#### Can you reproduce the problem? If so, describe how. If not, describe troubleshooting steps you took before opening the issue.

https://borgbackup.readthedocs.io/en/1.2.0/changes.html#version-1-2-0-2022-02-22-22-02-22

The sample commands (`--repair`, `compact --cleanup-commits`) **should** include `--` (double ASCII hyphen):

https://github.com/borgbackup/borg/blame/master/docs/changes.rst#L240
https://github.com/borgbackup/borg/blame/master/docs/changes.rst#L246

> run `borg compact --cleanup-commits` to clean up a ton of 17 bytes long files in your repo caused by a borg 1.1 bug

Instead, when displayed on the page (and therefore when copy/pasted) they are rendered with the double hyphen becoming a single Unicode en dash:

> run `borg compact –cleanup-commits` to clean up a ton of 17 bytes long files in your repo caused by a borg 1.1 bug

**Comment by ThomasWaldmann** [comment]
Yeah, that looks bad, it should keep `--` and not transform that.

**Comment by ThomasWaldmann** [comment]
https://www.sphinx-doc.org/en/master/usage/configuration.html?highlight=dash#confval-smartquotes

**Comment by ThomasWaldmann** [comment]
We currently have that in the config:
```
# If true, SmartyPants will be used to convert quotes and dashes to
# typographically correct entities.
html_use_smartypants = True
```

The sphinx version running at RTD is 1.8.6.

**Comment by ThomasWaldmann** [comment]
https://github.com/borgbackup/borg/commit/b08064bb4ed4bbafcd17281c6b5c92a63cd8a79d

Hmm, is our markup for options wrong? Shall we rather disable dashes to avoid such issues?

@enkore ^

**Comment by enkore** [comment]
Yes, because there are a lot of --option mentions all over the docs. Though we could also change the default role used for \`...\`, because rST thinks it's <cite> right now - hence the smartypants.

**Comment by ThomasWaldmann** [comment]
Fixed by #6348 #6350 #6351.
